### PR TITLE
Update readme for new release

### PR DESCRIPTION
**Pull request by lukeIam**
To be merged when the new homepage is live.

- Remove XP
- Add link to new homepage
- Changed download section
- Add badges

Suggestions welcome...

### CODE REVIEW

**Comment by flokuep** [comment]
We should mention that linux isn't _really_ supported by us, but could propably built (with some/a lot of work).

**Comment by lukeIam** [comment]
> We should mention that linux isn't really supported by us, but could propably built (with some/a lot of work).

We should also mention that on that on the homepage #240 .
(The main problem is not to build Vocaluxe, it's building the dependencies...)

**Comment by flokuep** [comment]
Done.

**Comment by flokuep** [comment]
We should update this readme, too: https://github.com/Vocaluxe/Vocaluxe/blob/develop/Output/Readme.txt

**Comment by hendrikbl** [comment]
We can also add some badges from [shields.io](https://shields.io/) to the readme.

E.g.:
[AppVeyor](https://ci.appveyor.com/project/lukeIam/vocaluxe)
`[AppVeyor](https://ci.appveyor.com/project/lukeIam/vocaluxe)`

[Travis branch](https://travis-ci.org/lukeIam/Vocaluxe/)
`[Travis branch](https://travis-ci.org/lukeIam/Vocaluxe/)`

[license](https://github.com/Vocaluxe/Vocaluxe/blob/develop/LICENSE.txt)
`[license](https://github.com/Vocaluxe/Vocaluxe/blob/develop/LICENSE.txt)`

[GitHub release](https://github.com/Vocaluxe/Vocaluxe/releases) **(for v4.0)**
`[GitHub release](https://github.com/Vocaluxe/Vocaluxe/releases)`

**Comment by lukeIam** [comment]
I already added some badges:
AppVeyor -> develop last build
Travis -> travis is currently not used (would be for linux builds)
license -> shild.io's license badge was really big, so I made a custom with
GPLv3
GitHub release -> badge for the latest release
- + Nightly
- + Open bug
- + Open feature requests


2017-08-28 14:57 GMT+02:00 Hendrik <notifications@github.com>:

**Comment by hendrikbl** [comment]
>I already added some badges:

Yeah, have seen it a second later :man_facepalming:

**Comment by flokuep** [comment]
I've updated missing roadmap and readme files. Please check and add some lines, if there is missing something.

File: Output/Readme.txt [hunk 1]
```diff
@@ -161,4 +167,4 @@ Song-DataBase (USDB):	http://usdb.animux.de/
 = 6. Source Code                =
 =================================
 Source Code (GitHub):	https://github.com/Vocaluxe/Vocaluxe
-SF.Net Page:			http://sourceforge.net/projects/vocaluxe/
\ No newline at end of file
+SF.Net Page:			http://sourceforge.net/projects/vocaluxe/
```
Comment (lukeIam) [inline Output/Readme.txt]: Should we remove SF here? (or at least add an outdated here?)

File: Output/Readme.txt [hunk 2]
```diff
@@ -23,7 +23,8 @@ Vocaluxe is an entirely new development in C#, inspired by the original UltraSta
 and the great Ultrastar Deluxe project.
 
 Supported Operating Systems / Requirements:
-- WinXP, Vista, Windows 7 or Windows 8 with .NET 4.0
+- Vista, Windows 7, Windows 8 or Windows 10 with .NET 4.0
```
Comment (lukeIam) [inline Output/Readme.txt]: Windows Vista

File: Output/Readme.txt [hunk 3]
```diff
@@ -23,7 +23,8 @@ Vocaluxe is an entirely new development in C#, inspired by the original UltraSta
...
+- Linux (no official support)
 - 1 GHz CPU, 512 MB RAM, Graphics card with OpenGL 2.1 or DirectX 9.0 support
 - Visual C++ Redistributable Packages 2008 and 2012
```
Comment (lukeIam) [inline Output/Readme.txt]: We need to update the "Visual C++ Redistributable Packages" section...
I guess only 2012 is required now - but should be tested.

File: Output/Readme.txt [hunk 4]
```diff
@@ -150,7 +156,7 @@ Sing screen
 =================================
 = 5. Help & Suppport            =
 =================================
-Bug Tracker:			http://84.200.73.138/redmine/projects/vocaluxe
+Bug Tracker:			https://github.com/Vocaluxe/Vocaluxe/issues
```
Comment (lukeIam) [inline Output/Readme.txt]: Vocaluxe Homepage: https://vocaluxe.org/

File: Output/Readme.txt (hunk 1 above)
Comment (flokuep) [reply Output/Readme.txt]: Thanks for your feedback. Regarding Visual C++ Redistributable Packages - Any idea how to test it?

**Comment by flokuep** [comment]
Thanks for your feedback. Regarding Visual C++ Redistributable Packages - Any idea how to test it?

**Comment by lukeIam** [comment]
> Regarding Visual C++ Redistributable Packages - Any idea how to test it?

I' will test it.
I'm currently setting up a clean win 10 in a vm...

**Comment by flokuep** [comment]
Anything to do here? Otherwise I would merge it.

**Comment by lukeIam** [comment]
> Anything to do here?

If we merge it now we may have some non working links on the front-page readme.md .
(they are for the new homepage which isn't live yet)

**Comment by flokuep** [comment]
Mhm. You're right. So what should we do? Update links?

**Comment by lukeIam** [comment]
> Mhm. You're right. So what should we do? Update links?

I would suggest: wait for the merge until the new page is ready (what's left there?)

**Pull request merged**